cdap-storage-ext-spanner/src/main/java/io/cdap/cdap/storage/spanner/compression/CompressionConfig.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2025 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.storage.spanner.compression;
+
+/**
+ * Configuration class for compression settings.
+ */
+public class CompressionConfig {
+
+  /**
+   * Suffix for compressed columns.
+   */
+  public static final String COMPRESSED_COLUMN_SUFFIX = "_compression_status";
+  /**
+   * Spanner limit for string field in bytes.
+   */
+  public static final int STRING_LIMIT = 2621440;
+  /**
+   * Spanner limit for byte[] field in bytes.
+   */
+  public static final int BYTES_LIMIT = 10 * 1024 * 1024;
```

Comment:
This is not the appropriate place to define these constants.

These are the limitations of Spanner instance in general and have nothing to do with compression. Can be defined in `SpannerStructuredTable` where they are used?